frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -129,6 +129,29 @@ function FileManagerMenu:setUpdateItemTable()
     self.menu_items.items = {
         text = _("Items"),
         sub_item_table = {
+            {
+                text = _("Files per page"),
+                help_text = _([[This sets the number of files and directories
+                per page in non-'classic' display modes.]]),
+                keep_menu_open = true,
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local curr_items = G_reader_settings:readSetting("files_per_page") or 10
```

Comment:
Yes you're right, it only applies to the detailed file list.
It seems to me that the four top items in "Other settings" only apply to the Detailed list - if that's the case, I'll add it there without the parenthesis. Maybe add a sub-menu?
![other-settings](https://user-images.githubusercontent.com/1787782/65823403-f6124200-e255-11e9-85ea-964943631abe.png)

Reply:
The `Show hints` must apply to both mosaic & detailed.
May be to give a small visual hint to what applies to what:
```
Show hint for book with description
Show hint for opened books in history
Series>
---------
(Detailed list) Files per page
Show number of page read
Show number of pages left to read
---------
(Classic mode) Show open in bold
---------
The others...
```
Just beware the following if you change the order:
https://github.com/koreader/koreader/blob/a934d2d52e45a52b2232689e04701beb7db5c4b6/plugins/coverbrowser.koplugin/main.lua#L422-L424

Reply:
Yes, saw it now. And only now that you mention it, I see the little double edge on the top right for descriptions. So everything still seems to work as it should.
From the screenshot above, I assume that the "insert generic items" still works as it should - I'll take your comment as a hint for future changes.

Reply:
Do you use a visual hint for menu items that open a dialog? In many apps it's three dots. In KOReader I haven't seen a visual hint, e.g. in "Items per page" or "Font size".
Is this something we should consider to introduce on a global level?

Reply:
I guess we have that inverted :)
Menu items that don't open anything have a checkbox on their left, and it just toggle the checkbox.
All others either open a dialog, or launch something (possibly a full screen dialog, like TOC).
(Three dots would just add a bit of unneeded noise, imho.)

Reply:
> Three dots would just add a bit of unneeded noise, imho.

The tl;dr is that I agree with that.

A summary of the much longer reply is that implementation of this feature (?) is quite inconsistent, and the various HIG/UI guidelines by, e.g., Apple, Microsoft, and GNOME provide neither rationale nor guidance that I've found. At most they imply it by using `Open...` as an example menu item.

Android and Sailfish don't use the principle. Microsoft doesn't tend to use it in ribbon and post-ribbon interfaces.

If we look at older Microsoft apps, the implementation seems oddly technical. I suspect that perhaps it quite simply is (i.e., auto-generated in some manner). Unlike the vast majority of implementations, older Microsoft apps are slightly more consistent. Items like the about dialog are `About...` and options is `Options...`. This is not the case in virtually all other apps. But even so, an item like `Help` doesn't typically have an ellipsis (or three dots). It may not technically be a dialog of the program, but the result is the same. It opens a different window/dialog as opposed to applying an action to the current window. 

You might think that perhaps it's MS that was inconsistent and those other apps have a different idea, for example *dialogs that apply an action to the current document*, but no one seems to have thought that much about it. As far as I can tell it's completely up in the air whether a dialog will be indicated and to what scope it applies.

That's a brief summary of my analysis of existing implementations from a few years ago, when I considered implementing `...`.

Of course I could've decided to do away with the inconsistencies in what seems to be the least bad implementation (by Microsoft), but it just didn't seem to *add* anything worthwhile. This menu item opens a dialog? Does it matter? The only truly relevant distinction is whether it applies to the currently opened document or to the entire program. Whether it opens a dialog to do so strikes me as an irrelevant detail.

PS If you've found anything different, please share your arguments or references. I don't intend this to shut down discussion, merely to indicate that I quite possibly gave it more thought than most.

Reply:
> Whether it opens a dialog to do so strikes me as an irrelevant detail.

Moreover, in our UI many actions apply either locally or as global defaults at the user's request. Tap for current document, long-press for global default.

Reply:
I was mainly wondering if it was a conscious decision or a case of "we haven't thought about it". Your answer is quite satisfactory, and I have no inclination to argue for a change.

With respect to "tap for current document", selecting a page in the file chooser is triggered with long-press, tap has no effect. It seems the wrong way around to me, unless you have reasons to keep it that way, I would kinda  like to change that.

Reply:
You'd have to check git blame to see if there's a particular reason. Perhaps it's for consistency with the skim dialog in the reader, which you can trigger by long pressing at the bottom. I don't think I ever gave the FileChooser controls much thought.

Reply:
I'll take that into a separate pull request.